### PR TITLE
Moving manage-cluster-templates.md into Manage Clusters

### LINKED ISSUE #404: Confusion around organization of content under "Authentication, Permissions, and Global Configurations"
From Slack:

> Any idea why this is below the authentication section in docs? [https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permiss[…]ons-and-global-configuration/manage-cluster-templates](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates#deploying-clusters-from-a-template-with-fleet) I would have expected this in an area about creating downstream clusters… but maybe we should split the chapter “authenticaion… global configuration” into two or three chapters?

Background context: based on previous feedback there was an effort to reduce the number of levels in the sidebar so that navigation would be easier for users. This may have resulted in new groupings that may not be as ntuitive, such as the example provided here.

### PR DESCRIPTION
**Pull request by sunilarjun**
Moving manage-cluster-templates.md into Manage Clusters subsection for more intuitive grouping, and cleaned up sidebars, redirect section, and Chinese translation. This PR fixes #404 .

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -415,6 +411,10 @@ module.exports = {
             to: '/how-to-guides/new-user-guides/manage-clusters/rotate-encryption-key',
             from: '/how-to-guides/advanced-user-guides/manage-clusters/rotate-encryption-key'
           },
+          {
+            to: '/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates',
+            from: '/how-to-guides/advanced-user-guides/manage-clusters/manage-cluster-templates'
```
Comment (btat) [inline docusaurus.config.js:416]:
```suggestion
            from: '/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates'
```

Suggestion made assuming the goal is the redirect from where this file was before it got moved/renamed as part of this PR. In any case, it doesn't look like the file was named/located at `/how-to-guides/advanced-user-guides/manage-clusters/manage-cluster-templates` at any point.

File: docusaurus.config.js [hunk 2]
```diff
@@ -335,10 +335,6 @@ module.exports = {
             to: '/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/example-use-cases',
             from: '/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/example-use-cases'
           },
-          {
-            to: '/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates',
-            from: '/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates'
-          },
```
Comment (btat) [inline docusaurus.config.js:341]: We probably should leave this block in for now until we decide on some guidelines on how we want to handle redirects in general. This redirect block and most of the ones surrounding it were added as part of the restructuring in https://github.com/rancher/rancher-docs/pull/234. 

Assuming it works then the full redirect chain (after combining with the one being added) would be

`/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates` --> `/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates` --> `/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates`

**Review by btat — CHANGES_REQUESTED** [review]

**Comment by sunilarjun** [comment]
Thanks for the review Billy! Pushed the proposed changes

**Comment by btat** [comment]
> Thanks for the review Billy! Pushed the proposed changes

Ah, looks like the build is failing because of the redirect block I suggested leaving in. That was the wrong suggestion, it should have been only keeping the `from` portion. Add the proper suggestion now.

File: docusaurus.config.js [hunk 3]
```diff
@@ -415,6 +415,10 @@ module.exports = {
             to: '/how-to-guides/new-user-guides/manage-clusters/rotate-encryption-key',
             from: '/how-to-guides/advanced-user-guides/manage-clusters/rotate-encryption-key'
           },
+          {
+            to: '/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates',
+            from: '/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates'
```
Comment (btat) [inline docusaurus.config.js:420]:
```suggestion
            from: ['/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates', '/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates']
```

**Comment by sunilarjun** [comment]
Gotcha just applied the new change, thanks for clarifying!

**Review by btat — APPROVED** [review]

**Pull request merged**